### PR TITLE
Logout float right in NavBar, Refresh auth token dynamically

### PR DESCRIPTION
**Pull request by piggy1-mvn**

### CODE REVIEW

**Comment by piggy1-mvn** [comment]
[Portal: Float Log Out Button to the right](https://trello.com/c/79INnoyG/80-portal-float-log-out-button-to-the-right)

File: src/components/common/Header.js [hunk 1]
```diff
@@ -1,6 +1,10 @@
 import React from "react";
 import { Link } from "react-router-dom";
-import { isAuthenticated, logout } from "../../helper/LoginHelper";
+import { logout } from "../../helper/LoginHelper";
+
+const rightStyle = {
```
Comment (diptyojha) [inline src/components/common/Header.js]: put it in index.css

File: src/components/common/Header.js [hunk 2]
```diff
@@ -1,6 +1,10 @@
...
+  float: 'right'
+}
```
Comment (diptyojha) [inline src/components/common/Header.js]: put this piece in index.css and call it here.

Comment (piggy1-mvn) [reply src/components/common/Header.js]: If you check the index.css, the default style for nav li is float left. This is for overriding the default style for one li element, which contains the Log Out link.

Comment (rajan123456) [reply src/components/common/Header.js]: If you check the index.css, the default style for nav li is float left. This is for overriding the default style for one li element, which contains the Log Out link.

**Comment by piggy1-mvn** [comment]
[Portal: Refresh Token dynamically](https://trello.com/c/QN5r7dCP/85-portal-refresh-token-dynamically)

**Comment by piggy1-mvn** [comment]
[Portal: Refresh Token dynamically](https://trello.com/c/QN5r7dCP/85-portal-refresh-token-dynamically)

**Comment by piggy1-mvn** [comment]
[Portal: Refresh Token dynamically](https://trello.com/c/QN5r7dCP/85-portal-refresh-token-dynamically)

**Pull request closed without merging**